### DISCUSSION TITLE
Not CD, Blu-ray, DVD or Data. Bailing out on sr0

### DISCUSSION THREAD

**Discussion by Ted-Corrrell** [automatic-ripping-machine/automatic-ripping-machine#892] [Q&A]
Hello, 

I have ripped a number of DVDs so far, but all of a sudden it won't even start a job. The log shows the following: 
Tue Jul 11 04:29:43 UTC 2023 Entering docker wrapper
Tue Jul 11 04:29:43 UTC 2023 [ARM] Not CD, Blu-ray, DVD or Data. Bailing out on sr0
But I have inserted a DVD. 
I am running the latest docker container (2.6.46) pulled just now. Before I was using 2.6.40, and receiving the same log message. 

It reacts each time the drive is closed and the disc will spin up. I set the log level to debug, and that didn't add anything to the log output. I have wiped all the files and recreated the container; no changes. 

Is there anything I can try?

**Comment by Ted-Corrrell** [comment]
Update: I just tried to rip a disc that I had already ripped, so I know it should work. When I inserted the disc, I got a pop-up notification from gnome [Movie] and giving me an option to open with Videos. The other movies I've tried haven't done that. 
It seems to be that older movies are more likely to work, and newer ones are more likely to not work. 
So it looks like it is an OS level issue. Any ideas on how to troubleshoot or diagnose the cause of an intermittent DVD not showing up?

**Comment by Ted-Corrrell** [comment]
It is probably a hardware issue. My computer is running Debian 11. I know Ubuntu is the only supported native OS, but I am using docker. I tried Debian 12, older DVDs still worked  and newer DVDs didn't. Then I tried Windows, and it still couldn't read newer DVDs. So I'll need to get a newer DVD drive, or maybe even a Blu-ray drive.

**Comment by mmorley0395** [comment]
i've got this issue too, running the latest docker image but the few dvds i've tried have failed with this error
